instrumentation/http/lib/opentelemetry/instrumentation/http/patches/client.rb

```diff
@@ -23,7 +23,7 @@ def perform(req, options)
               'net.peer.port' => uri.port
             }.merge!(OpenTelemetry::Common::HTTP::ClientContext.attributes)
 
-            tracer.in_span("HTTP #{request_method}", attributes: attributes, kind: :client) do |span|
+            tracer.in_span("HTTP #{request_method} #{uri.path}", attributes: attributes, kind: :client) do |span|
```

Comment:
This change is in contradiction with the opentelemetry specification for HTTP Span Names. https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/http/#name

> HTTP client spans have no http.route attribute since client-side instrumentation is not generally aware of the “route”, and therefore HTTP client spans SHOULD use {http.method}. Instrumentation MUST NOT default to using URI path as span name, but MAY provide hooks to allow custom logic to override the default span name.

So we can't accept this change upstream.